relations/routes/tests/test_api.py

```diff
@@ -0,0 +1,125 @@
+"""Tests for :mod:`routes.api`."""
+
+import json
+import os
+from http import HTTPStatus
+from typing import Any, Optional
+from unittest import TestCase, mock
+from datetime import datetime
+
+import jsonschema
+from flask import Flask
+
+from arxiv.users.helpers import generate_token
+from arxiv.users import auth
+from arxiv.users.domain import Scope
+from relations.factory import create_api_app
+from .. import api
+
+class TestExternalAPIRoutes(TestCase):
+    """Sample tests for external API routes."""
+
+    def setUp(self) -> None:
+        """Initialize the Flask application, and get a client for testing."""
+        os.environ['JWT_SECRET'] = 'foosecret'
+        self.app = create_api_app()
+        self.client = self.app.test_client()
+
+    @mock.patch(f'{api.__name__}.controllers.create_new')
+    def test_create_new(self, mock_create_new: Any) -> None:
```

Comment:
Totally fine to annotate tests, but not necessarily required. When we do static checks e.g. on Travis we usually ``| grep -v "test.*"``